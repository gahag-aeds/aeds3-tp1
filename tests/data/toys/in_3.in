4
0.0000 0.75674 0.3573 0.24971 
0.24326 0.0000 0.72357 0.41531 
0.6427 0.27643 0.0000 0.82172 
0.75029 0.58469 0.17828 0.0000 
