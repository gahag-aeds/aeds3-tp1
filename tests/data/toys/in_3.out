0.203114 0.230260 0.335256 0.231370 